src/amber/cli/commands/database.cr

```diff
@@ -23,7 +23,20 @@ module Amber::CLI
       end
 
       class Help
-        caption "# Performs database maintenance tasks"
+        header <<-EOS
+          Performs database migrations and maintenance tasks. Powered by micrate (https://github.com/juanedi/micrate)
```

Comment:
Does it make sense to say "Powered by micrate" here? What if we change the underlying library?
 I think the fact that we're using micrate is an implementation detail.

Thoughts?